genomel/slurm/postgres/status.py

```diff
@@ -0,0 +1,134 @@
+'''
+Postgres tables for the PDC CWL Workflow
+'''
+from sqlalchemy.orm import sessionmaker, mapper
+from sqlalchemy import MetaData, Table
+from sqlalchemy import Column, String
+import re
+
+
+class Metrics(object):
+    pass
+
+
+class BamFiles(object):
+    pass
+
+
+class FastqFiles(object):
+    pass
+
+# Organize rows of genomel_fastq_input to dictionary
+
+
+def retrive_reads(cases):
+    s = dict()
+    for row in cases:
+        s.setdefault(row.aliquot, {})
+        s[row.aliquot].setdefault(row.read_group, {})
```

Comment:
it's hard for me to get these `read_group` ids when they are keys along with others like `project`, `s3_xxx`. it's better to construct dict like ```{'read_group_1': {'read_group_id': 'xxx', 'input_id_r1': 'xxx'}}``` Or, you move other keys under `read_group`, so that the dict is ```{'aliquot_id': {'read_group_id': {XXX}}}```